frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-add-options.feature

```diff
@@ -110,19 +110,46 @@ Feature: Create Pipeline from Add Options
 
 
         #Marking this as manual due to git rate limit  issue
-        @manual
-        Scenario Outline: Add Pipeline display in git workload for builder image: P-01-TC08
+        @regression @manual
+        Scenario Outline: Add Pipeline option display in git workload for builder image: P-01-TC08
             Given user is at Import from Git form
              When user enters Git Repo url as "<git_url>"
              Then Add pipeline checkbox is displayed
 
         Examples:
-                  | git_url                                                   |
-                  | https://github.com/sclorg/dancer-ex.git                   |
-                  | https://github.com/sclorg/cakephp-ex.git                  |
-                  | https://github.com/redhat-developer/s2i-dotnetcore-ex.git |
-                  | https://github.com/sclorg/golang-ex.git                   |
-                  | https://github.com/sclorg/ruby-ex.git                     |
-                  | https://github.com/sclorg/django-ex.git                   |
-                  | https://github.com/spring-projects/spring-boot            |
-                  | https://github.com/sclorg/nodejs-ex.git                   |
+                  | git_url                                        |
+                  | https://github.com/sclorg/dancer-ex.git        |
+                  | https://github.com/sclorg/cakephp-ex.git       |
+                  | https://github.com/sclorg/golang-ex.git        |
+                  | https://github.com/sclorg/ruby-ex.git          |
+                  | https://github.com/sclorg/django-ex.git        |
+                  | https://github.com/spring-projects/spring-boot |
+                  | https://github.com/sclorg/nodejs-ex.git        |
+
+
+        @regression @manual
+        Scenario Outline: Add Pipeline option display for dotnet builder image with context directory": P-01-TC09
```

Comment:
Technically, Pipeline option will be displayed if the templates are available for selected builderImage + resourceType combination, it has nothing to do with context dir, we could remove this scenario, It is not needed.

Reply:
yes, but this dotnet builder image is not covered in earlier scenario right, so I think better to be include this scenario